rollup.config.js

```diff
@@ -38,6 +41,24 @@ module.exports = [
     },
     external
   },
+  // js
```

Comment:
this is not needed anymore, right?

Reply:
yes @kurkle, I had added if we wanted to "mitigate" the break... I'll remove it. No hurry, right?  it needs major version, correct?

Reply:
Removed it